<?xml version='1.0'?>
<launch>
	<node pkg="ml_detector" type="ml_detector_node" name="ml_detector" clear_params="true" output="screen">
		<!-- System Parameters -->
		<param name="system/publish_overlay" value="true" type="boolean"/>
		<param name="system/send_detailed_tag_info" value="true" type="boolean"/>
		<param name="system/show_rejected" value="false" type="boolean"/>
		<param name="system/refine_strategy" value="true" type="boolean"/>
		<param name="system/camera_is_rectified" value="false" type="boolean"/>

		<!-- Detection Parameters /-->
		<rosparam command="load" file="$(find ml_detector)/launch/detect_config.yaml"/>
		<rosparam command="load" file="$(find ml_detector)/launch/board_config.yaml"/>

		<!-- Marker Parameters -->
		<remap from="~detected_markers" to="/ml_landmarks/detected_markers"/>

		<!-- Image Parameters -->
		<remap from="~camera_info" to="/webcam/camera_info"/>
		<remap from="~image_raw" to="/webcam/image_raw"/>
		<!--param name="image_transport" value="compressed"/-->
	</node>
</launch>
